src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -1976,53 +1976,78 @@ limitations under the License.
             <include cluster="Media Playback" client="false" server="true" clientLocked="false" serverLocked="true"></include>
             <include cluster="Keypad Input" client="false" server="true" clientLocked="false" serverLocked="true"></include>
             <include cluster="Application Launcher" client="false" server="true" clientLocked="false" serverLocked="false"></include>
-            <include cluster="Media Input" client="false" server="true" clientLocked="false" serverLocked="false"></include>
-            <include cluster="On/Off" client="false" server="true" clientLocked="false" serverLocked="false"></include>
+            <include cluster="Media Input" client="false" server="true" clientLocked="false" serverLocked="true"></include>
```

Comment:
Why is "Media Input" `serverLocked="True"`?  It's not mandatory for this device type.

Please check all the serverLocked bits carefully.

Reply:
If I'm getting the spec correctly It actually depends if the device has physical inputs or not.
Since all example apps are simulating input presence, it makes sense to have this value set to true.

Reply:
This is not about example apps.  This file is configuring ZAP UI for anyone trying to build a Matter application.  Having `serverLocked="true"` here means developers will get errors when they try to turn off this cluster, which they have to be able to do if they have no physical inputs.

Reply:
Updated.